Xsup 50462 carbon black (#39964)
demisto/content: Packs/CarbonBlackEnterpriseEDR/ReleaseNotes/1_1_42.md

#### Integrations

##### Carbon Black Enterprise EDR

- Updated the Docker image to: *demisto/python3:3.12.8.3296088*.

> - Fixed an issue where the fetch command could incorrectly return zero events even when incidents were available.

Reviewer: ```suggestion
- Fixed an issue where the ***fetch-incident*** command incorrectly calculated the limit.
```
